.github/workflows/mt.yml

```diff
@@ -10,10 +10,6 @@ jobs:
   tests:
     runs-on: ubuntu-latest
 
-    strategy:
```

Comment:
I have no reason to think that mutation test results would vary based on PHP version.